php/common/crud.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+  session_start();
+
+  $request = $_REQUEST['type'];
+
+  if(isset($_REQUEST['id'])) {
```

Comment:
Don't do this. You would have to do an isset for ALL  transactions concerning common pages at this rate. For each function, example retrieveTitle, we know that if its going to be this function that's called, then the $_REQUEST['id'] will ALWAYS be set. So we can just get rid of the isset and diretso na $category_id = $_REQUEST['id'] inside the retrieveTitle function because we're very much sure that it is set.